packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationForm.tsx

```diff
@@ -89,13 +90,15 @@ export const CreateOrganizationForm = withCardStateProvider((props: CreateOrgani
       lastCreatedOrganizationRef.current = organization;
       await setActive({ organization });
 
-      void userMemberships.revalidate?.();
-
       if (sessionTasksContext) {
-        await sessionTasksContext.nextTask();
+        await clerk.__internal_navigateToTaskIfAvailable({
+          redirectUrlComplete: sessionTasksContext.redirectUrlComplete,
+        });
         return;
       }
 
+      void userMemberships.revalidate?.();
```

Comment:
Previously, I've rewritten the entire `OrganizationList` logic within `TaskSelectOrganization` to avoid a flicker of UI on revalidation of memberships. 

As I started to export `TaskSelectOrganization` as a public component, I realized that it couldn't receive a custom appearance for `OrganizationList`. 

Simpler solution is just to not revalidate the data in place when resolving a `select-organization` task.